public/nativedisplay.h

```diff
@@ -186,6 +186,26 @@ class NativeDisplay {
   virtual void SetBrightness(uint32_t /*red*/, uint32_t /*green*/,
                              uint32_t /*blue*/) {
   }
+
+ /**
+  * API for setting video color in HWC
+  */
+  virtual void SetVideoColor(HWCColorControl /*color*/, float /*value*/) {
+  }
```

Comment:
When these are set, do we than always need to do postprocessing of video buffers using libva or can display handle any of these things directly ?

Reply:
when this is set, we have to use va for postprocessing. display can't handle these directly.

Reply:
k thank we need to have a state in DisplayQueue indicating this. Than if we have a video buffer in planemanager we need to set state needing processing.